pkg/backup/item_collector.go

```diff
@@ -60,17 +169,22 @@ type kubernetesResource struct {
 
 // getItemsFromResourceIdentifiers converts ResourceIdentifiers to
 // kubernetesResources
-func (r *itemCollector) getItemsFromResourceIdentifiers(resourceIDs []velero.ResourceIdentifier) []*kubernetesResource {
+func (r *itemCollector) getItemsFromResourceIdentifiers(
+	resourceIDs []velero.ResourceIdentifier,
+) []*kubernetesResource {
 	grResourceIDsMap := make(map[schema.GroupResource][]velero.ResourceIdentifier)
 	for _, resourceID := range resourceIDs {
-		grResourceIDsMap[resourceID.GroupResource] = append(grResourceIDsMap[resourceID.GroupResource], resourceID)
+		grResourceIDsMap[resourceID.GroupResource] = append(
+			grResourceIDsMap[resourceID.GroupResource], resourceID)
 	}
 	return r.getItems(grResourceIDsMap)
 }
 
 // getAllItems gets all relevant items from all API groups.
```

Comment:
Please clarify in the comment that this does not collect ALL items, but the filtered items.

Reply:
Modified.